### ISSUE TITLE
Installed properly on EC2 Server, now what?

### ISSUE THREAD

**Issue by shaykahen** [CartoDB/cartodb#2800]
So I am completely new to this stuff, so please forgive me for anything that may seem silly to the experienced user.

I have nearly installed all the required files on an EC2 ubuntu server via ssh. My question is, once that is complete, how can I access the dashboard or cartoDB interface in general? Pretend for a moment, my public ip address is 1.1.1.1, how do I access cartodb?

**Comment by xavijam** [comment]
There has been no activity on this issue for more several months. We are closing it. If you think this still needs to be addressed please open a new issue.
